includes/utilities.inc

```diff
@@ -387,3 +390,62 @@ function islandora_solr_islandora_basic_collection_backend_callable($collection_
     array_map($map_to_pids, $qp->islandoraSolrResult['response']['objects']),
   );
 };
+
+/**
+ * Truncate the field display based on entire field result(s).
+ *
+ *
+ * @param array $display_values
+ *   An array of the values that are to be processed for truncation.
+ * @param int $max_length
+ *   The maxium length of characters to display before truncating results.
+ * @param bool $add_ellipsis
+ *   Boolean to enable the display of an ellispsis.
+ * @param bool $word_safe
+ *   Boolean to enable word safe in truncate_utf8.
+ * @param int $wordsafe_length
+ *   Min wordsafe length.
+ * @param string $separator
+ *   A separator to use for display output.
+ *
+ * @return string
+ *   The updated display values.
+ */
+function islandora_solr_truncate_field_display($display_values, $max_length, $add_ellipsis, $word_safe, $wordsafe_length, $separator) {
+  if (count($display_values) > 0) {
+    $mod_path = drupal_get_path('module', 'islandora_solr_metadata');
```

Comment:
Missed this last pass: Should move the JS into this module. (since `islandora_solr_metadata` is already dependent on `islandora_solr`... Don't want to introduce circular dependencies).